assets/speakers.json

```diff
@@ -1,15 +1,15 @@
 [
   {
-    "lightning": true,
+    "lightning": false,
     "free": true,
-    "time": "6:30 PM"
+    "time": "7:00 PM"
   },
   {
-    "avatar": "/static/Johannes.jpg",
-    "link": "https://twitter.com/frostney_",
-    "name": "Johannes Stein",
-    "title": "Let's develop a React-powered game!",
-    "text": "This talk is about React-powered game development where Johannes will be live coding a PIRATE game throughout his talk. With each example he will be implementing a new feature towards a playable prototype.",
+    "avatar": "/static/Mario.jpg",
+    "link": "https://twitter.com/realmario",
+    "name": "Mario",
```

Comment:
Mario Rogic * (according to Linked IN)

Reply:
We should probably just ask quickly to be sure :) it’s a name afterall